catalog/utilities/media_props_gen/generate_media_properties.py

```diff
@@ -1,27 +1,36 @@
 """Automatic media properties generation."""
 
 import logging
+import sys
 from dataclasses import dataclass
 from pathlib import Path
 
-from column_parser import parse_python_columns
-from db import create_db_props_dict
-from md import Md
 
-from common.constants import MEDIA_TYPES
+# Avoid import error in tests
+sys.path.insert(0, str(Path(__file__).parents[3]))
```

Comment:
This feels like it might not be advisable...is it necessary? I know we did **not** have to do something similar for the DAG documentation generation 🤔

Reply:
DAG documentation is all in one file.
Here, I broke the code into several files. Without this, there's a "Module not found" error when testing. 
Could we somehow update the path when testing so we don't need this line?

Reply:
I tried experimenting with this, and realized we could make _huge_ improvements here by deleting one (1) file 😆: https://github.com/WordPress/openverse/pull/4029

If you're able to rebase your PR once this is merged, this should prevent having to alter the system path like this at all!